src/pages/trade/LaunchableMarket.tsx

```diff
@@ -0,0 +1,205 @@
+import { useMemo, useRef, useState } from 'react';
+
+import { useMatch } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+
+import { TradeLayouts } from '@/constants/layout';
+import { AppRoute } from '@/constants/routes';
+
+import { useBreakpoints } from '@/hooks/useBreakpoints';
+
+import breakpoints from '@/styles/breakpoints';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { DetachedSection } from '@/components/ContentSection';
+import { AccountInfo } from '@/views/AccountInfo';
+import { LaunchMarketSidePanel } from '@/views/LaunchMarketSidePanel';
+
+import { useAppSelector } from '@/state/appTypes';
+import { getSelectedTradeLayout } from '@/state/layoutSelectors';
+
+import { getDisplayableTickerFromMarket } from '@/lib/assetUtils';
+
+import { HorizontalPanel } from './HorizontalPanel';
+import { InnerPanel } from './InnerPanel';
+import { MarketSelectorAndStats } from './MarketSelectorAndStats';
+import { MobileBottomPanel } from './MobileBottomPanel';
+import { MobileTopPanel } from './MobileTopPanel';
+import { TradeHeaderMobile } from './TradeHeaderMobile';
+
+const LaunchableMarket = () => {
+  const tradePageRef = useRef<HTMLDivElement>(null);
+  const { isTablet } = useBreakpoints();
+  const tradeLayout = useAppSelector(getSelectedTradeLayout);
+  const match = useMatch(`/${AppRoute.Trade}/:marketId`);
```

Comment:
meganit: maybe this can be returned via `useCurrentMarketId`

Reply:
Good idea. Will add in my follow up